ops/ansible/playbooks-ccs/files/client_data_server_ab2d_prod_certificate.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN CERTIFICATE-----
+MIIEsjCCApoCCQCG3rsaL5VtKTANBgkqhkiG9w0BAQsFADAbMRkwFwYDVQQDDBBh
+YjJkLXByb2QtY2xpZW50MB4XDTIwMDQxMzE4MjkwNVoXDTIxMDQxMzE4MjkwNVow
+GzEZMBcGA1UEAwwQYWIyZC1wcm9kLWNsaWVudDCCAiIwDQYJKoZIhvcNAQEBBQAD
+ggIPADCCAgoCggIBALhcRrXWBXp6CQlip7iXaF+tuvs7IHV6zVlN4wVIxgoDytVX
+xo/tPxnL9Bvx4xcawecx1TRuCe75i4iAKaIupzx7fVjuQDJh7eYe1HCgUfwMMN6c
+5ZbE7KrL7U84vHwIpKETwsucFClzqKDoqhZ4aGfD/KIUhIgFLFzLkKLGTnuYuadC
++tu8t7/T9CALonPRQczIY82n4+0VoRFtklBWMDss/l3LqWnW8oUDKLbdnhG40Nii
+DfIgiv+pYLuByRxIXLQUPW3WbtWCVp+U6ybf9YER8jtzJL7iSO5pY6sqU2/0WFVF
+X2KhdVcCjabGxO8YY28ZrqlBAyYa1dA6U+G7upaFaD5Em7ZXr9hRSEq03oCuMIMG
+L1wCUuv1aw6pFWfEW+TyYFpR7A8HksCi/sNkab75ULRrILIssmtmmLXVMhXzzK1X
+FAIcvs4EJCVEruiFh88MaP5sThBcYUheS4r0USg+TltuPGa1wkAhyIsJYgNKNVD2
+gyV8efQI9NBTg/sTcmxwt92+ML16eZE+3umusozvXT174TCZov73VuihZnzmLF4R
+0Av6sSsxL/CUOoRMh0v0KPKpZbS52Tonwi8spdGoeSlgIUG3V90/PGRwUNTSclqc
+knK4ZRYlKh5+/wR2H5JGaFiOIPJhcxOQWZYLEHNHoksOgDOtLvereb0zwArRAgMB
+AAEwDQYJKoZIhvcNAQELBQADggIBAHPos/yQGZN4hvx3UZHINUC48TinYn4MNjjl
+/htFpQrk5i2ooKXOZGm+SFkMuTq/r46SFDzpQtg2uFWkiXVopzyPb2APP1dEjG9T
+V+/t/5R2klAcqiQ7bzweyOF/B5Ct69/dWfzxmyeLLOjSGpZRVo0Zqx0JB6RQmR+m
+mMDnsEdkXtQBSwpd/u7r3qaHR/RvibJMjAOJW2iGhsc2776/nJDuzyrX8RZn4wTo
+ADhaz8Y5JHcEznHSVYvEoEXGXx4xXv8U6yQZmk2x0xzQ5fB674/uuZFWfi7GMnco
+bC04DyCJUA07Rupgk6nCoSVdoOS0zSZ5RpYSEmokkABeqH3UcDbVmgRkhjFXFTLk
+Y4nUJknE+L4OkmggI2MyIa0Dn1m0mSx3BFC5r2LvyXiYIMxJeuxD+ohZtCH++0QB
+qyRWAPaKLkNTnhJgefp5kC95l3IFr+kfAi+ub3M0V8laarprucaXUgIpAFjHlb65
+/AG92G4FQx4NrWt511HiDqYdNzt+0eJ8sSIk5AuZNkLq2XFtP8j3rTOzT3edu+Ih
+vejQJ8SmpO7mJOo320q1hBEI4LsUfdbAufQQWd2Ti9lnjfIO4qwFsAdnBTBGbKLY
+GkJFhMc0aQjhUhsaG5v7bv5V55IPc0ns/NeroI1Cr4UStKbMYlw89ztLLVF2cPtt
+MXIEdz5t
+-----END CERTIFICATE-----
```

Comment:
I'm not able to dump this certificate by hand. 
```
openssl x509 -in ~/projects/temp/AB2D.pem  -text -noout                                                                                                                                                 09:26:27
unable to load certificate
4604417644:error:09FFF066:PEM routines:CRYPTO_internal:bad end line:/BuildRoot/Library/Caches/com.apple.xbs/Sources/libressl/libressl-22.260.1/libressl-2.6/crypto/pem/pem_lib.c:791:
```

Just to double check my command line, I ran it against a known good cert. 
```
openssl x509 -in ~/projects/beneficiary-fhir-data/ops/ansible/playbooks-ccs/files/client_data_server_ab2d_sbx_certificate.pem  -text -noout                                                             09:26:28
Certificate:
    Data:
        Version: 1 (0x0)
        Serial Number: 17737621763724056112 (0xf628b10423029630)
    Signature Algorithm: sha256WithRSAEncryption
```

Reply:
Hi @RickHawesUSDS :
`bash-3.2$ curl -o ~/Temp/AB2D.pem https://raw.githubusercontent.com/CMSgov/beneficiary-fhir-data/f3df72b64ef1ec6a8cdbb48c16ffe4ac28557fde/ops/ansible/playbooks-ccs/files/client_data_server_ab2d_prod_certificate.pem; openssl x509 -in ~/Temp/AB2D.pem  -text -noout`

```
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1688  100  1688    0     0   3059      0 --:--:-- --:--:-- --:--:--  3063
Certificate:
    Data:
        Version: 1 (0x0)
        Serial Number: 9718410767053974825 (0x86debb1a2f956d29)
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN=ab2d-prod-client
```

🤷‍♂️

Reply:
@RickHawesUSDS and @dkrylovsb Cert looks good on my end. 

```jzmbp:tmp johnzulim$ openssl x509 -in client_data_server_ab2d_prod_certificate.pem -text -noout```
```Certificate:
    Data:
        Version: 1 (0x0)
        Serial Number:
            86:de:bb:1a:2f:95:6d:29
        Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN = ab2d-prod-client
        Validity
            Not Before: Apr 13 18:29:05 2020 GMT
            Not After : Apr 13 18:29:05 2021 GMT
        Subject: CN = ab2d-prod-client
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                RSA Public-Key: (4096 bit)
                Modulus:```